### ISSUE TITLE
`config.yml` should be namespaced and abide by the XDG base directory specification

### ISSUE THREAD

**Issue by Fyorl** [foundryvtt/foundryvtt-cli#4]
At the moment it's stored at the top level of `%appdata%` on Windows, or in `$HOME` on Linux. `config.yml` is a very generic name to have sitting at the top level of those directories, it could belong to almost any program, so I think it would make sense if it were either placed under something like a `fvtt/` directory, or if it used some other config filename conventions like `.fvttrc.yml`.

Moreover, on Linux it should abide by the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and be stored under `$XDG_DATA_HOME`, or fall back to `$HOME/.local/share` (with above namespacing considerations), resorting to storing the file in `$HOME` only if all else fails. Basically the same rules that Foundry VTT itself uses in order to determine the user data path.

**Comment by FloRad** [comment]
Mind you that as far as I can tell the ENV variable `%XDG_DATA_HOME%` does not exist on windows so something like
```TS
  process.env.APPDATA || (process.platform == 'darwin' ? process.env.HOME + '/Library/Preferences': process.env.HOME +'/.local/share');
```

would probably be ideal to determine a base path

**Comment by cswendrowski** [comment]
Closed as of v0.0.7
